### DISCUSSION TITLE
page web within the markdown page

### DISCUSSION THREAD

**Discussion by pirvana** [requarks/wiki#3350] [Help / Questions]
**Question**

Good, a query, before the update 2.0.12, when making the code 

```
<object data="https://access.redhat.com/solutions/3025421" type="" align="center" width="100%" style="height:9999px">
</object>
```

 showed the website, and currently with the same code no longer shows me anything, just a white background, there would be another way to integrate a page web within the markdown page

**Host Info (please complete the following information):**
OS: [e.g. Centos 7, Docker]
Wiki.js version: [e.g. 2.0.12]
Database engine: [e.g. postgres 9.7]

**Comment by ssimontis** [comment]
The HTML renderer responsible for whitelisting HTML tags for display no longer allows the <object> tag to be used on a site. You can see a list of all allowed tags under server/rendering/html-security/renderer.js.

**Comment by ssimontis** [comment]
@NGPixel can we please close this one out since it is working as intended?
